.github/workflows/duplicate_pr.yml

```diff
@@ -0,0 +1,71 @@
+name: Duplicate PR
+
+#Duplication workflow must be run before PR branch is deleted
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  duplicate_pr:
+    name: Duplicate PR
+    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body,'duplicate this PR') }}
+    runs-on: ubuntu-latest
+    steps:
+      # https://github.com/marketplace/actions/github-script
+      - name: Get PR Intial Commit SHA
+        if: ${{ github.event.issue.pull_request.merged_at }}
+        id: initial-sha
+        uses: actions/github-script@0.9.0
+        with:
+          script: |
+            const sha = context.payload.comment.body.split(" ").pop()
```

Comment:
Should this do any validation on the SHA before proceeding?

Reply:
I was considering that, but I think it's appropriate to just pass it into the script and it'll fail on the script side if the SHA is invalid.

Or are you asking from a security angle?